src/index.md

```diff
@@ -22,9 +22,9 @@ Welcome to the official documentation of Core, the community-focused game creati
 ## Organization of the Documentation
 
 - [**Getting Started**](editor_intro.md) explains the essential information  you need to start using Core, from [installation](installing_core.md) to [making a game](my_first_multiplayer_game.md).
-- [**Tutorials**](tutorials/overview.md) contains step-by-step instructions to make specific features or full games. It includes an [introduction to Lua scripting in Core](lua_basics_lightbulb.md), and a collection of [examples](examples.md) to learn from and use.
+- [**Tutorials**](tutorials/overview.md) contains step-by-step instructions to make specific features or full games. It includes an [introduction to Lua scripting in Core](lua_basics_lightbulb.md), and a collection of [examples](api/index.md) to learn from and use.
```

Comment:
@slinkous remove the examples part maybe?

Reply:
Agree to removing the examples part.